cmd/start.go

```diff
@@ -115,21 +116,24 @@ func start(cmd *cobra.Command) error {
 		return fmt.Errorf("error initializing beacon: [%v]", err)
 	}
 
+	initializeMetrics(ctx, clientConfig, netProvider, blockCounter)
+	registry := initializeDiagnostics(clientConfig)
+	diagnostics.RegisterConnectedPeersSource(registry, netProvider, signing)
+	diagnostics.RegisterClientInfoSource(registry, netProvider, signing, build.Version, build.Revision)
```

Comment:
Instead of passing `registry` as the first argument, can we call these functions on the registry pointer? This would allow us to avoid the weird construction with passing `commonDiagnostics` package. 

We would introduce `Registry` structure inside `diagnostics` package that would wrap `Registry` from `keep-common` and that would expose `RegisterConnectedPeersSource`, `RegisterClientInfoSource`, `RegisterApplicationSource`.

Reply:
yea, I like it [3987347](https://github.com/keep-network/keep-core/pull/3240/commits/3987347511864a4001e8d8323da1c5097aabcd5d)